musica/util.F90

```diff
@@ -0,0 +1,38 @@
+! Copyright (C) 2024 National Center for Atmospheric Research,
+! SPDX-License-Identifier: Apache-2.0
+module musica_ccpp_util
+
+  implicit none
+
+  private
+  public :: has_error_occurred
+
+contains
+
+  !> @brief Evaluate a MUSICA error for failure and convert to CCPP error data
+  !> @param[in] error The error code to evaluate and convert.
+  !> @param[out] error_code The CCPP error code.
+  !> @param[out] error_message The CCPP error message.
+  !> @return True for an error, false for success.
+  logical function has_error_occurred(error, error_code, error_message)
+    use musica_util, only : error_t
+    type(error_t), intent(in) :: error
+    integer, intent(out) :: error_code
+    character(len=512), intent(out) :: error_message
+
+    character(len=30) :: error_code_str
+
+    if ( error%is_success( ) ) then
+      error_code = 0
+      error_message = ''
+      has_error_occurred = .false.
+      return
+    end if
+    error_code = error%code( )
+    write(error_code_str, '(I30)') error%code( )
+    error_message = '[MUSICA Error]: ' // error%category( ) // '[' // &
+                    trim( adjustl( error_code_str ) ) // ']: ' // error%message( )
+    has_error_occurred = .true.
+  end function has_error_occurred
+
+end module musica_ccpp_util
```

Comment:
github is indicating that there is no "newline" at the end of this file.   stackoverlow indicates that best practice is a newline should be present on the last line of a file.

Reply:
done!

Reply:
This solved the problem for this particular file.  The newline issue occurs on most (but not all) of the files in this PR.  Let me know if you don't see the red circle with the dash in it, and I can indicated each file with the problem

Reply:
I think they should all now have newlines at the end. Let me know if I missed any!